commcare_export/data_types.py

```diff
@@ -0,0 +1,24 @@
+import sqlalchemy
+
+DATA_TYPE_TEXT = 'text'
+DATA_TYPE_BOOLEAN = 'boolean'
+DATA_TYPE_DATE = 'date'
+DATA_TYPE_DATETIME = 'datetime'
+DATA_TYPE_INTEGER = 'integer'
+
+DATA_TYPES_TO_SQLALCHEMY_TYPES = {
+    DATA_TYPE_BOOLEAN: sqlalchemy.Boolean(),
+    DATA_TYPE_DATETIME: sqlalchemy.DateTime(),
+    DATA_TYPE_DATE: sqlalchemy.Date(),
+    DATA_TYPE_INTEGER: sqlalchemy.Integer(),
+}
+
+class UnknownDataType(Exception):
+    pass
+
+
+def get_sqlalchemy_type(data_type):
+    if data_type not in DATA_TYPES_TO_SQLALCHEMY_TYPES:
+        raise UnknownDataType()
```

Comment:
This seems handy for debugging
```suggestion
        raise UnknownDataType(data_type)
```